### ISSUE TITLE
Composer require command gives error

### ISSUE THREAD

**Issue by ressw** [noxzemart/rt-instagram-sdk-php#1]
Hello,

Sorry for the trouble and stupid question, I just started learning PHP. I wanted to install your package "noxzemart/rt-instagram-sdk-php", but the command "composer require noxzemart/rt-instagram-sdk-php" gives me the error "InvalidArgumentException] Could not find a matching version of package noxzemart/rt-instagram-sdk-php. Check the package spelling, your version constraint and that the package is available in a stability which matches your minimum-stability (stable)." Could you tell me how to solve this problem? I will be glad of any of your answers. 

thanks!

**Comment by noxzemart** [comment]
Hello

what PHP framework are you use ?

**Comment by ressw** [comment]
without framework, server on local machine. Exploring the possibilities of programmatically working with Instagram, running scripts through the console or IDE

**Comment by noxzemart** [comment]
> without framework, server on local machine. Exploring the possibilities of programmatically working with Instagram, running scripts through the console or IDE

ok so please try from this packagist
https://packagist.org/packages/liamcottle/instagram-sdk-php or https://packagist.org/packages/bloolight/instagram-sdk-php
it is same.

**Comment by ressw** [comment]
thanks for your answer and help,
trying to solve the problem myself, I found and tested many similar packages at https://packagist.org/, but they all throw errors with old dependencies (I tried using the --with-all-dependencies switch, but the problem may not be solved). Of all these packages, your updates are the newest, so I disturbed you, I apologize.

**Comment by ressw** [comment]
I downloaded the source_code archive, and a little later I will try to add your package to autoload from composer manually.

**Comment by ressw** [comment]
I managed. I added your package folder to the 'vendor' directory, removed the composer.lock file, added the package name with version "^1.0" to the root composer.json file. But most importantly, I manually set the version number to low for all the dependent packages already installed. For example, netresearch/jsonmapper needed version 1.0, and I had 4.0. And then the command "composer update" completed everything successfully, updates were downloaded, dependencies were linked and a new composer.lock file was created. The problem has been resolved. Further it will be very interesting to try to work with your package.

I want to say thank you very much for your response and desire to help.
